src/visualization/utils/useStaticLegend.ts

```diff
@@ -122,11 +123,14 @@ export const useStaticLegend = (properties): StaticLegendConfig => {
             estimatedHeight = length * sampleMaxHeight + padding
           }
 
-          const updatedRatio = estimatedHeight / totalHeight
-          const heightRatio =
-            updatedRatio <= STATIC_LEGEND_HEIGHT_RATIO_MAXIMUM
-              ? updatedRatio
-              : STATIC_LEGEND_HEIGHT_RATIO_MAXIMUM
+          let heightRatio = estimatedHeight / totalHeight
```

Comment:
do we need a divide by zero check here, just in case `totalHeight` ever ends up being `0`?

Reply:
Yes good catch. In fact, we have to guard against all sorts of silly falsy values.

Reply:
Updated.